public/electron.js

```diff
@@ -255,6 +275,15 @@ ipc.on('error', (event, args) => {
   if (opt === 0) app.quit()
 })
 
+ipc.on('save-config', (event, config, participantID, studyID) => {
+  if (savePath === "") {
+    savePath = getSavePath(participantID, studyID)
```

Comment:
the other ones of these also check for participant and studyID, but this one doesn't - is that because those variables are passed in?  should that check maybe move into that function?